src/imgui/csvparser.hpp

```diff
@@ -0,0 +1,187 @@
+#ifndef CSV_PARSER_H
+#define CSV_PARSER_H
```

Comment:
```suggestion
#ifndef CASH_OVERFLOW_CSV_PARSER_HPP
#define CASH_OVERFLOW_CSV_PARSER_HPP
```

Reply:
also the name of file could be `csvParser.hpp` since the other filenames are using camelcase.